assets/src/blocks/ShareButtons/EmailShare/EmailShareFrontend.js

```diff
@@ -0,0 +1,32 @@
+import { SvgIcon } from "../SvgIcon";
+import { parseUtmParams } from '../utils';
+
+const { __ } = wp.i18n;
+
+export const EmailShareFrontend = ({
+  url,
+  utmMedium,
+  utmContent,
+  utmCampaign,
+  gaEvent,
+  gaEventCategory,
+  gaCategory,
+  gaAction,
+  gaLabel,
+  openInNewTab,
+  title,
+  body,
+}) => (
+  <a href={ `mailto:?subject=${title}&body=${body ? encodeURIComponent(body) : ''}\n${url}&${parseUtmParams('email', utmMedium, utmContent, utmCampaign)}` }
+    className="social-share-button email"
+    data-ga-event={gaEvent}
+    data-ga-event-category={gaEventCategory}
+    data-ga-category={gaCategory}
+    data-ga-action={gaAction}
+    data-ga-label={gaLabel}
+    { ...openInNewTab && { target: '_blank' } }
+  >
+    <SvgIcon {...{name: "envelope"}} />
+    <span className="visually-hidden">{__( 'Share via', 'planet4-master-theme' )} Email</span>
```

Comment:
We should make the entire string translatable: `"Share via Email"`. I don't think all languages use the word "email", and it might not come at the end either.

Reply:
The text domain should be `planet4-blocks-backend` for admin strings in the plugin..